hydromt_wflow/workflows/river.py

```diff
@@ -372,23 +372,27 @@ def river_floodplain_volume(
     flood_depths = np.atleast_1d(flood_depths).ravel().astype(dtype)  # force 1D
     _, ucat_vol = flwdir.ucat_volume(idxs_out=idxs_out, hand=hand, depths=flood_depths)
     # force minimum volume based on river width * length
-    min_vol = (
-        ds_model["rivwth"].values[None, ...]
-        * ds_model["rivlen"].values[None, ...]
-        * flood_depths[:, None, None]
-    )
-    ucat_vol = (
-        np.where(riv_mask, np.maximum(ucat_vol, min_vol), -9999)
-        .round(0)
-    )
+    rivlen = np.maximum(ds_model["rivlen"].values, 1)  # avoid zero division errors
+    min_vol = ds_model["rivwth"].values * rivlen * flood_depths[0]
+    ucat_vol[0, :, :] = np.maximum(ucat_vol[0, :, :], min_vol).round(0)
```

Comment:
I think it is probably better to leave out the rounding (`.round(0)`) in lines 377 and line 382. Without the rounding the provided flood volume error becomes a lot smaller. In Wflow.jl it is then possible to raise a warning based on volume (instead of width), which is more consistent (per river cell). I did test this and without the rounding it is fine to use of a volume threshold of 0.01 m3, without raising a warning. See also https://github.com/Deltares/Wflow.jl/pull/233/commits/f21fb7f9ecaa09e0d54a4a9843d3f2858a13f619.

Reply:
OK, will change this!